x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/total_feature_importance_summary/feature_importance_summary.tsx

```diff
@@ -0,0 +1,233 @@
+        classificationType = 'binary_classification';
+        sortedData = (totalFeatureImportance as ClassificationTotalFeatureImportance[]).map((d) => {
+          return {
+            featureName: d.feature_name,
+            // in case of binary classification, both classes will have the same mean importance
+            meanImportance: d.classes[0].importance.mean_magnitude,
+          };
+        });
+      }
+
+      // if multiclass classification
+      // stack them in order of increasing importance
+      // so for each feature, biggest importance on the left to smallest importance on the right
+      if (totalFeatureImportance[0].classes.length > 2) {
+        classificationType = 'multiclass_classification';
+
+        (totalFeatureImportance as ClassificationTotalFeatureImportance[]).forEach((feature) => {
+          feature.classes
+            .sort((a, b) => a.importance.mean_magnitude - b.importance.mean_magnitude)
+            .forEach((featureClass) => {
+              sortedData.push({
+                featureName: feature.feature_name,
+                meanImportance: featureClass.importance.mean_magnitude,
+                className: featureClass.class_name,
+              });
+            });
+        });
+        _barSeriesSpec = {
+          xAccessor: 'featureName',
+          yAccessors: ['meanImportance'],
+          splitSeriesAccessors: ['className'],
+          stackAccessors: ['featureName'],
+        };
+      }
+    }
+    // if regression
+    if (isRegressionTotalFeatureImportance(totalFeatureImportance[0])) {
+      classificationType = 'regression';
+
+      sortedData = (totalFeatureImportance as RegressionTotalFeatureImportance[]).map(
+        (d: RegressionTotalFeatureImportance) => ({
+          featureName: d.feature_name,
+          meanImportance: d.importance.mean_magnitude,
+        })
+      );
+    }
+
+    // sort from largest importance at top to smallest importance at bottom
+    sortedData = sortedData.sort((a, b) => b.meanImportance - a.meanImportance);
+
+    // only show legend if it's a multiclass
+    const _showLegend = classificationType === 'multiclass_classification';
+    const _chartHeight =
+      totalFeatureImportance.length * (totalFeatureImportance.length < 5 ? 40 : 20) + 50;
+    return [sortedData, _barSeriesSpec, _showLegend, _chartHeight];
+  }, [totalFeatureImportance]);
+
+  const { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } = docLinks;
+  const tickFormatter = useCallback((d) => Number(d.toPrecision(3)).toString(), []);
+
+  return (
+    <EuiPanel>
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false}>
+          <EuiTitle size="xs">
+            <span>
+              <FormattedMessage
+                id="xpack.ml.dataframe.analytics.exploration.featureImportanceSummaryTitle"
+                defaultMessage="Feature importance summary"
+              />
+            </span>
+          </EuiTitle>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
```

Comment:
The styling of this help icon should be consistent with that used for `pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx` I think. Currently the spacing and the vertical alignment is different between the two panels:

![image](https://user-images.githubusercontent.com/7405507/94415453-74f5cf80-0175-11eb-8317-31c71fcf4782.png)

Reply:
Updated here https://github.com/elastic/kibana/pull/78238/commits/89f1ee34ba1f6884c1012b2d08e19044dce6a5cd